Cue/app/common/CueDrawerLayout.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+var React = require('React');
+var DrawerLayoutAndroid = require('DrawerLayoutAndroid');
+
+class CueDrawerLayout extends React.Component {
+  _drawer: ?DrawerLayoutAndroid;
+
+  constructor(props: any, context: any) {
+    super(props, context);
+
+    (this: any).openDrawer = this.openDrawer.bind(this);
+    (this: any).closeDrawer = this.closeDrawer.bind(this);
+    (this: any).onDrawerOpen = this.onDrawerOpen.bind(this);
+    (this: any).onDrawerClose = this.onDrawerClose.bind(this);
+    (this: any).handleBackButton = this.handleBackButton.bind(this);
+  }
+
+  render() {
+    const {drawerPosition, ...props} = this.props;
+    const {Right, Left} = DrawerLayoutAndroid.positions;
```

Comment:
I think we can remove, don't need to have it since the menu will always be the same on the left